cli/pkg/cmdutil/check.go

```diff
@@ -6,6 +6,12 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	ErrNotAuthenticated = fmt.Errorf("not authenticated, please run 'rill login'")
+	ErrNoOrganization   = fmt.Errorf("no organization is set, pass `--org` or run `rill org switch`")
+)
```

Comment:
Can we keep the inline errors? Generally we only extract errors to variables if a) they are returned from multiple functions, or b) we need to be able to check them using `errors.Is` somewhere upstream. But neither of those conditions apply here.

Reply:
Fair enough 👍